### PR TITLE
Support summarizing non kubernetes clients.

### PR DESCRIPTION
**Pull request by fejta**
Also stop assuming an extra column exists, even when processing a kubernetes client.

/assign @michelle192837 @chases2

### CODE REVIEW

File: pkg/summarizer/summary.go [hunk 1]
```diff
@@ -531,7 +531,7 @@ func statusMessage(cols int, rows []*statepb.Row, recent int) string {
 const noGreens = "no recent greens"
 
 // latestGreen finds the ID for the most recent column with all passing rows.
-func latestGreen(grid *statepb.Grid) string {
+func latestGreen(grid *statepb.Grid, firstExtraColumn bool) string {
```
Comment (michelle192837) [inline pkg/summarizer/summary.go]: Parameter name is still confusing to me, maybe something like 'useFirstExtra'?

Comment (fejta) [reply pkg/summarizer/summary.go]: Fixed

**Comment by fejta** [comment]
Now with passing unit tests??

**Pull request merged**